sdk/core/core-auth/src/tokenCredential.ts

```diff
@@ -52,6 +52,11 @@ export interface GetTokenOptions {
    * Allows specifying a tenantId. Useful to handle challenges that provide tenant Id hints.
    */
   tenantId?: string;
+
+  /**
+   * Claim details to perform the Continuous Access Evaluation authentication flow
+   */
+  claims?: string;
```

Comment:
These are the claims we get from the challenge right? We are already passing it in  `authorizeRequestOnClaimChallenge` callback but forgot to update this interface when the callback was originally added

Reply:
Yes exactly! This parameter is already passed in [code](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/core/core-client/src/authorizeRequestOnClaimChallenge.ts#L87)

Reply:
Sure will do it